pkgs/development/libraries/libevent/default.nix

```diff
@@ -11,13 +16,20 @@ stdenv.mkDerivation rec {
 
   # libevent_openssl is moved into its own output, so that openssl isn't present
   # in the default closure.
-  outputs = [ "out" "dev" "openssl" ];
+  outputs = [ "out" "dev" ]
+    ++ stdenv.lib.optional enableOpenssl "openssl"
+    ;
   outputBin = "dev";
-  propagatedBuildOutputs = [ "out" "openssl" ];
+  propagatedBuildOutputs = [ "out" ]
+    ++ stdenv.lib.optional enableOpenssl "openssl"
+    ;
 
-  buildInputs = [ openssl ] ++ stdenv.lib.optional stdenv.isCygwin findutils;
+  buildInputs = []
+    ++ stdenv.lib.optional enableOpenssl "openssl"
```

Comment:
openssl shouldn't be a string here but the reference to the package. :wink:

Reply:
Oops, you can tell I never actually tried this with `openssl ` enabled...thanks for catching that.